recipe/0001-inject-cmake-project.patch

```diff
@@ -64,6 +64,9 @@ index 0000000..c15df47
 +                            $<INSTALL_INTERFACE:include>
 +                           )
 +
++target_compile_definitions(getopt PRIVATE EXPORTS_GETOPT _CONSOLE)
```

Comment:
What is `_CONSOLE` for?

Reply:
I actually don't know. I am merely emulating the behavior over in the original getopt.vcxproj https://github.com/libimobiledevice-win32/getopt/blob/master/getopt.vcxproj#L83.

Since this switches away from that and I am unfamiliar with this code, I am happy to forward what was there before.

Reply:
Please add a comment explaining this at the top of the file with a link.

Reply:
Following up, `_CONSOLE` appears to be a MSVC convention for labelling programs as commandline oriented. The definition itself doesn't appear in `getopt.h` or `getopt.c` but may appear in the MSVC implementations of the c standard libraries, so best to leave it around.